src/android/Library/src/MultiImageChooserActivity.java

```diff
@@ -401,6 +407,33 @@ public SquareImageView(Context context) {
         public void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
             super.onMeasure(widthMeasureSpec, widthMeasureSpec);
         }
+
+        @Override
+        protected void onDraw(Canvas canvas) {
+            super.onDraw(canvas);
+
+            Paint iconPaint = new Paint();
+            iconPaint.setTypeface(mDonkeyFont);
+            float size = this.getWidth() * 0.25f;
+            float padding = this.getWidth() * 0.05f;
+            iconPaint.setTextSize(size);
+
+            if(this.checked) {
+                int zenputBlue = getResources().getColor(R.color.zenputBlue);
+                iconPaint.setColor(zenputBlue);
+                iconPaint.setAlpha(255);
+                Paint circlePaint = new Paint();
+                circlePaint.setTypeface(mDonkeyFont);
+                circlePaint.setColor(Color.WHITE);
+                circlePaint.setTextSize(size);
+                canvas.drawText("\uE06E", padding, padding+size, circlePaint);
```

Comment:
This is fine for now, but future idea: I wonder if we could make a script that converts our donkeyfont.css file into a Java/Obj. C class that we could import and lets us reference these icons by a class name. For example:

```java
mDonkeyFont = Donkeyfont.getTypeface();
```

And:

```java
canvas.drawText(Donkeyfont.ROTATE_1, padding, padding+size, circlePaint);
```

That probably wouldn't be hard, and it would make our code a bit cleaner. It would also remove the danger of accidentally changing the character code for any of our icons.